docs/hardware/toonv1.md

```diff
@@ -0,0 +1,245 @@
+* SPI serial EEPROM
+* 256 Kbit (32 KiB x 8 bits)
+* Address: spi0.0
+* Linux driver: `at25`
+
+This EEPROM is likely not supposed to be accessed directly, but rather through
+the Z-Wave controller's interface. However, the SPI bus is accessible from
+Linux, and the EEPROM can be accessed directly.
+
+[Datasheet](./datasheets/Fujitsu-MB85RS256TY.pdf) of a compatible part.
+
+### USB2
+Microchip USB3370 (x2)
+* Enhanced Single Supply Hi-Speed USB ULPI Transceiver (USB PHY)
+* Used to connect the USB port and the Wi-Fi module
+* Connected to the SoC's USB2 ULPI interface
+
+[Datasheet](./datasheets/Microchip-USB3370.pdf)
+
+
+### OpenTherm / power port
+Toon v1 has a 2-pin port on its back which is used to connect the Toon to a
+central heating system or boiler, as well as to provide power.
+
+The pins match up with a connector embedded in the wall mount.
+
+OpenTherm communications are done through UART1 (ttymxc1), at 4800,8n1 (4800
+baud(?), 8 data bits, no parity, 1 stop bit).
+
+#### Ketelmodule
+Commonly, the wall mount connector is connected to a Ketelmodule (boiler
+module). This module is a separate device, and allows the Toon to communicate
+with the boiler as well as provide adequate power to the Toon.
+
+This module is not required for the Toon to function. You can also connect the
+wall mount connector directly to 24V DC power, and use the Toon without
+OpenTherm functionality.
+
+### JTAG header
+The Toon v1 PCB has a 14-pin JTAG header. The kind folks at Quby have even
+populated the header with pins, which makes it super easy to connect a JTAG
+debugger or Raspberry Pi.
+
+Its pinout is as follows:
+
+Pin | Signal | Description
+--- | ---    | ---
+1   | RTCK   | Return clock signal
+2   | TRST   | Test reset
+3   | GND    | Ground
+4   | TCK    | Test clock
+5   | GND    | Ground
+6   | TMS    | Test mode select
+7   | SRST   | System reset
+8   | TDI    | Test data in
+9   | Vt     | ?
+10  | TDO    | Test data out
+11  | RxD    | UART Receive data
+12  | (NC)   | Not connected
+13  | TxD    | UART Transmit data
+14  | GND    | Ground
+
+* Pin 1 is the top left pin when orienting the PCB with the ethernet port at
+  the bottom left.
+* Pin 2 is the top right pin.
+* The TxD and RxD pins are connected to UART0 (ttymxc0), which is used for
+  console output by U-Boot and Linux.
+
+```plaintext
+          1___
+RTCK 01 - |.|.| - 02 TRST
+ GND 03 - |.|.| - 04 TCK
+ GND 05 - |.|.| - 06 TMS
+SRST 07 - |.|.| - 08 TDI
+  Vt 09 - |.|.| - 10 TDO
+ RxD 11 - |.|.| - 12 NC
+ TxD 13 - |.|.| - 14 GND
+```
+## Sensors
+
+### temperature sensor
+Analog Devices ADT7410
+* I2C temperature sensor
+* Accuracy: ±0.5°C from -40°C to +105°C
+* 16-bit resolution (~0.0078°C)
+* Address: 0x49 @ i2c-0
+* Linux driver: `adt7410`
+
+Note: the ATD7410 would normally reside on 0x48, and while there is a driver
+configured for this address, it does not appear to work.
+
+Potentially, the device was (planned to be) moved to 0x48 on later hardware
```

Comment:
or maybe have two? I don't know. Could also note here that the Toon frontend software appears to apply some offset because the device heats itself a bit.